frontend/src/app/shared/custom/scoring/scoring.component.html

```diff
@@ -1,16 +1,14 @@
 <div class="d-flex flex-column">
   <div class="scoring-container d-flex flex-row">
     <div class="flex-fill">
-      <div>
-        <span class="okr-score-label">
-          <span class="text-center text-small">Fail</span>
-        </span>
-        <span class="okr-score-label okr-score-commit">
-          <span class="text-center text-small">Commit</span>
-        </span>
-        <span class="okr-score-label">
-          <span class="text-center text-small">Target</span>
-        </span>
+      <div class="okr-score-label">
+        <h5 class="text-center text-small">Fail</h5>
```

Comment:
do we really need the parent div around each element?

Reply:
Having wrappers may unfortunately be mandatory here. I will replace them with a `span` since they have `inline-grid` styling applied.